disasterinfosite/settings.py

```diff
@@ -109,3 +109,6 @@
 
 ### ^^^^^^^^^^^^^^^^^^^^^^^^^ ###
 ### END HEROKU CONFIGURATIONS ###
+
+MEDIA_ROOT = os.path.join(BASE_DIR, 'static', 'img')
+MEDIA_URL = 'static/img/'
```

Comment:
This is going to require a corresponding edit on the deployed servers.  Not a problem - I'm just mentioning it so you can help me not forget.